Bunkum.Listener/Request/ListenerContext.cs

```diff
@@ -119,7 +119,7 @@ public virtual long ContentLength
     /// Writes a byte array to the response stream.
     /// </summary>
     /// <param name="buffer">The byte array to write.</param>
-    public void Write(byte[] buffer)
+    public void Write(ReadOnlySpan<byte> buffer)
```

Comment:
No concerns about breaking changes here? Dunno about edge cases, should we have a `byte[]` overload?